### PR TITLE
feat(content): add Republic Navy Advisory System and add related humanitarian missions news

### PR DESCRIPTION
**Pull request by dorbarker**
## Summary
Adds `news` for the "Republic Navy Advisory System", as suggested in #8819.

The `news` is broken into two parts. One that is always available on Republic planets, and another that is available during the FW war.

### CODE REVIEW

File: data/human/news.txt
Comment (lumbar527) [inline data/human/news.txt:1831]: I don't think merchants are any less frequent in the South during the war; the FW doesn't attack them.

Comment (dorbarker) [reply data/human/news.txt:1831]: To me this is [a tension between lore and mechanics](https://en.wikipedia.org/wiki/Ludonarrative_dissonance). You're completely right that merchants don't become less frequent, but what government wouldn't at least attempt to prevent trade with their enemy? Besides, you don't know what the origin of any given merchant is - maybe they're from the appropriate side of the frontlines?

Comment (lumbar527) [reply data/human/news.txt:1831]: I guess, how does this effect merchants?

Comment (dorbarker) [reply data/human/news.txt:1833]: There is no mechanical effect on merchant behaviour. It's pure flavour text building upon other lore already within the game.

If there is a strong preference for having the `news` match the ship frequencies , then I'm okay modifying or removing these lines to suit.

Comment (lumbar527) [reply data/human/news.txt:1833]: It's probably fine, I just wanted to note it.

**Review by roadrunner56 — COMMENTED** [review]
Good start, but I would beef this up a little with more possible messages. #8819 mentioned environmental disasters like earthquakes and the like, maybe you could add more messages about those?

File: data/human/news.txt
Comment (lumbar527) [reply data/human/news.txt:1812]:
```suggestion
			"Please be advised: a travel advisory remains in effect for "
		word
			"Albatross"
			"Bloodsea"
			"Buccaneer Bay"
			"Covert"
			"Deadman's Cove"
			"Freedom"
			"Greenrock"
			"Haven"
			"New Tortuga"
			"Smuggler's Den"
			"Stormhold"
			"Thule"
			"Zenith"
		word
			"."
```
Here's a list.

**Review by W1zrad — CHANGES_REQUESTED** [review]
Looks good! Re: the topic of Republic/Navy and merchant travel to the FW, I think having them discourage travel is good; saying it's `prohibited` I'm not too big a fan of. Based on that, I'd recommend amending line 1832, but 1833 is fine as-is.

**Comment by dorbarker** [comment]
> Looks good! Re: the topic of Republic/Navy and merchant travel to the FW, I think having them discourage travel is good; saying it's `prohibited` I'm not too big a fan of. Based on that, I'd recommend amending line 1832, but 1833 is fine as-is.

> Good start, but I would beef this up a little with more possible messages. #8819 mentioned environmental disasters like earthquakes and the like, maybe you could add more messages about those?

Sounds good. Will implement both these changes as soon as I am closer to a real keyboard

File: data/human/news.txt
Comment (lumbar527) [inline data/human/news.txt:1866]:
```suggestion
		word
			"."
		word
			""
			"Please notify Republic officials immediately if you have been detained by any ship not granted legal authority to do so by the Republic Navy."
```
Otherwise you get something like `...for ZenithPlease...`

File: data/human/news.txt
Comment (lumbar527) [inline data/human/news.txt:1868]:
```suggestion
			" Please notify Republic officials immediately if you have been detained by any ship not granted legal authority to do so by the Republic Navy."
			" Report all sightings of pirate vessels immediately."
			" The following message is a test. The following message is a test. The following message is a test: Radiation hazard. Evacuate immediately. Radiation hazard. Evacuate immediately. Radiation hazard. Evacuate immediately."
```

**Review by lumbar527 — COMMENTED** [review]
Stuff related to my hail change. (#8873)

File: data/human/human missions.txt
Comment (roadrunner56) [inline data/human/human missions.txt]:
```suggestion
		government "Republic" "Syndicate" "Independent" "Neutral"
```

File: data/human/human missions.txt
Comment (roadrunner56) [inline data/human/human missions.txt]:
```suggestion
		government "Republic" "Syndicate" "Independent" "Neutral"
```

**Review by roadrunner56 — CHANGES_REQUESTED** [review]

File: data/human/human missions.txt
Comment (lumbar527) [reply data/human/human missions.txt]: I think this has been fixed already—https://github.com/endless-sky/endless-sky/pull/8871/commits/e0787b2f16ec3e7936cc46f37974ffbeeee8c95e

**Comment by dorbarker** [comment]
Currently failing CI tests, though should be resolved upon a successful merge of #8880

File: data/human/news.txt
Comment (dorbarker) [reply data/human/news.txt:1866]: This will end up with the travel advisory adjoined to, for example, an unrelated systems test of the radiation alert. I expect this will get worse in the future as more messages are added. I think the better way of handling this is to refactor the travel advisory into its own `phrase` so it doesn't interfere with other messages.

**Comment by dorbarker** [comment]
> Good start, but I would beef this up a little with more possible messages. #8819 mentioned environmental disasters like earthquakes and the like, maybe you could add more messages about those?

I have added a mission that responds to volcanic and cold snap events. I would like to have similar earthquakes and other natural disasters, but that will require adding new attributes to planets. I've done that with `frozen` in #8880, but will hold off on adding the necessary `tectonic` attributes until this is merged. I also think adding forest fire missions is pretty straightforward via the existing `forest` attribute.

There's no payment for these missions, at present. I wrote them for the RPG value as a volunteer job with no remuneration, but this is changeable if desired.

**Comment by lumbar527** [comment]
I'd say a slight payment would be good, perhaps from grateful people you helped.

**Comment by dorbarker** [comment]
> I'd say a slight payment would be good, perhaps from grateful people you helped.

Yeah. Like maybe with some flavour text saying it covers "fuel and crew expenses" or something?

**Review by lumbar527 — APPROVED** [review]
Getting better, just some `substitution` things.

File: data/human/news.txt
Comment (roadrunner56) [inline data/human/news.txt:1906]: You could replace `Radiation hazard` with a list of different types of hazards to reduce repetition.

**Review by roadrunner56 — CHANGES_REQUESTED** [review]

File: data/human/news.txt
Comment (dorbarker) [reply data/human/news.txt]: Good idea. Should be resolved by 302bd37

**Review by roadrunner56 — CHANGES_REQUESTED** [review]

**Review by roadrunner56 — APPROVED** [review]
This is an excellent lore addition, thanks for taking the time to write up something like this.

**Comment by dorbarker** [comment]
I had to change the emergency alert test. As written, it was doing things like:
`Radiation hazard. Evacuate immediately. Cryogenic fluid release. Flee immediately. Meteor incoming. Seek shelter.`

I wasn't able to get `replace` or `substitutions` to pick from a `phrase`, so for the moment, I have brute-forced the warnings that RNAS gives during its test.

File: data/human/human missions.txt
Comment (bene-dictator) [inline data/human/human missions.txt:6174]: Two things:
- There is already a wildfire job, but we can either remove that one (because I certainly like linking it to the Navy) or limit it only to the Free Worlds. You can look at that job's payment and use it here.
- These missions would be better in `jobs.txt`.

File: data/human/news.txt
Comment (bene-dictator) [inline data/human/news.txt:1920]: Might also add a mention to the Alphas.

**Review by bene-dictator — COMMENTED** [review]
This is very good!

File: data/human/news.txt
Comment (dorbarker) [reply data/human/news.txt:1920]: Good idea.  I will write up a new `news` node that only shows once the Alphas appear

File: data/human/human missions.txt
Comment (dorbarker) [reply data/human/human missions.txt:6174]: Will do. Also, looking at them now, I think I should probably consolidate "volcanic aid" and "wildfire aid" commodities into single "flames aid".  Their contents are basically identical.

File: data/human/jobs.txt
Comment (bene-dictator) [inline data/human/jobs.txt:3788]:
```suggestion
	description "Deliver humanitarian aid of <cargo> to <destination> by <date> following a severe cold snap. Payment is <payment>."
```

File: data/human/jobs.txt
Comment (bene-dictator) [inline data/human/jobs.txt:3816]:
```suggestion
	description "Deliver humanitarian aid of <cargo> to <destination> by <date> following a severe volcanic eruption. Payment is <payment>."
```

File: data/human/jobs.txt
Comment (bene-dictator) [inline data/human/jobs.txt:3823]: The way you have these missions set up, you can get unnecessarily long trips to your destination. One possible mission might have a player go from Wayfarer to Calda and and get only 8000 credits for it. You should consider adding a distance filter up to 8 systems away.
```suggestion
		government "Republic" "Syndicate" "Independent" "Neutral"
		distance 1 8
```

File: data/human/jobs.txt
Comment (bene-dictator) [inline data/human/jobs.txt:3849]:
```suggestion
		government "Republic" "Syndicate" "Independent" "Neutral"
		distance 1 8
```

File: data/human/jobs.txt
Comment (bene-dictator) [inline data/human/jobs.txt:3875]:
```suggestion
		government "Republic" "Syndicate" "Independent" "Neutral"
		distance 1 8
```

File: data/human/jobs.txt
Comment (bene-dictator) [inline data/human/jobs.txt:3795]:
```suggestion
		government "Republic" "Syndicate" "Independent" "Neutral"
		distance 1 8
```

File: data/human/jobs.txt
Comment (bene-dictator) [inline data/human/jobs.txt:3793]:
```suggestion
		government "Republic" "Syndicate"
```
Syndicate planets are part of the Republic too and get Navy protection, so they'd presumably help out.

File: data/human/jobs.txt
Comment (bene-dictator) [inline data/human/jobs.txt:3821]:
```suggestion
		government "Republic" "Syndicate"
```

File: data/human/jobs.txt
Comment (bene-dictator) [inline data/human/jobs.txt:3842]:
```suggestion
	description "Deliver humanitarian aid of <cargo> to <destination> by <date> following a severe wildfire. Payment is <payment>."
```

File: data/human/jobs.txt
Comment (bene-dictator) [inline data/human/jobs.txt:3868]:
```suggestion
	description "Transport <bunks> firefighters to <planet> by <date>. Payment is <payment>."
```

File: data/human/jobs.txt
Comment (bene-dictator) [inline data/human/jobs.txt:3873]:
```suggestion
		government "Republic" "Syndicate"
```

File: data/human/jobs.txt
Comment (bene-dictator) [inline data/human/jobs.txt:3812]:
```suggestion
		dialog `Shivering, yet appreciative aid workers welcome your shipment. You look on, feeling satisfied that <cargo> will soon arrive to those that need it most. Back in your ship, you're notified that you've recieved <payment> for your service.`
```

File: data/human/jobs.txt
Comment (bene-dictator) [inline data/human/jobs.txt:3838]:
```suggestion
		dialog `Relieved-looking aid workers welcome your shipment. You look on, feeling satisfied that <cargo> will soon arrive to those that need it most. Back in your ship, you're notified that you've recieved <payment> for your service.`
```
(Consider changing this suggestion to something more appropriate)

File: data/human/jobs.txt
Comment (bene-dictator) [inline data/human/jobs.txt:3864]:
```suggestion
		dialog `Ash-covered, but relieved aid workers welcome your shipment. You look on, feeling satisfied that <cargo> will soon arrive to those that need it most. Back in your ship, you're notified that you've recieved <payment> for your service.`
```
Ditto

File: data/human/news.txt
Comment (lumbar527) [inline data/human/news.txt:1865]:
```suggestion
	phrase
		"pirate worlds"
```

File: data/human/news.txt
Comment (lumbar527) [inline data/human/news.txt:1855]:
```suggestion
	phrase
		"pirate worlds"
```

File: data/human/news.txt
Comment (lumbar527) [inline data/human/news.txt:1875]: Does it really involve navigation? I would say it's more like a pirate hazard.

File: data/human/news.txt
Comment (lumbar527) [inline data/human/news.txt:1879]:
```suggestion
	phrase
		"pirate worlds"
```

File: data/human/news.txt
Comment (lumbar527) [inline data/human/news.txt:1893]:
```suggestion
	phrase
		"rnas travel advisory [1]"
		"rnas travel advisory [2]"
		"rnas travel advisory [3]"
```

**Review by lumbar527 — COMMENTED** [review]
Mostly that you can say `phrase ""` instead of `word "${}"`

File: data/human/news.txt
Comment (dorbarker) [reply data/human/news.txt:1875]: I was trying to mimic the slightly stilted manner in which government alerts are often written. Happy to change it though if it seems odd.

File: data/human/jobs.txt
Comment (dorbarker) [reply data/human/jobs.txt:2709]: Fixed

File: data/human/jobs.txt
Comment (bene-dictator) [inline data/human/jobs.txt]:
```suggestion
		dialog `Relieved-looking aid workers welcome your shipment. You look on, feeling satisfied that <cargo> will soon arrive to those that need it most. Back in your ship, you're notified that you've received <payment> for your service.`
```
Oops.

File: data/human/jobs.txt
Comment (bene-dictator) [inline data/human/jobs.txt]:
```suggestion
		dialog `Shivering, yet appreciative aid workers welcome your shipment. You look on, feeling satisfied that <cargo> will soon arrive to those that need it most. Back in your ship, you're notified that you've received <payment> for your service.`
```

File: data/human/jobs.txt
Comment (bene-dictator) [inline data/human/jobs.txt]:
```suggestion
		dialog `Ash-covered, but relieved aid workers welcome your shipment. You look on, feeling satisfied that <cargo> will soon arrive to those that need it most. Back in your ship, you're notified that you've received <payment> for your service.`
```

**Review by bene-dictator — CHANGES_REQUESTED** [review]
Sorry I have bad spelling : )

**Review by W1zrad — CHANGES_REQUESTED** [review]
Sorry for the wait! I think we're good to go after these changes.

Also, for future reference, updating the PR title when the scope of the PR changes (adding the humanitarian aid missions) is always appreciated :)

**Review by bene-dictator — APPROVED** [review]

**Review by W1zrad — APPROVED** [review]
Good to merge!

**Pull request merged**
